test/.ocamlformat

```diff
@@ -0,0 +1 @@
+parse-docstrings = false
```

Comment:
This seems too specific, the same problem will happen for any other option. Perhaps set profile = default instead?